### PR TITLE
Bug/post merge fixes

### PR DESCRIPTION
**Pull request by RyanJarv**
This fixes a few issues that came up after merging https://github.com/RhinoSecurityLabs/pacu/pull/247.

### CODE REVIEW

File: .github/workflows/python-deploy.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - uses: actions/checkout@v2
```
Comment (RyanJarv) [inline .github/workflows/python-deploy.yml:18]: Git repo won't be checked out if this isn't included.

File: pacu/main.py [hunk 2]
```diff
@@ -197,7 +197,7 @@ def log_error(self, text, exception_info=None, session=None, local_data=None, gl
             if session:
                 log_file_path = '{}/error_log.txt'.format(session_dir())
             else:
-                log_file_path = 'global_error_log.txt'
+                log_file_path = '{}/global_error_log.txt'.format(session_dir())
```
Comment (RyanJarv) [inline pacu/main.py:200]: Global error log wasn't getting written out.

File: pacu/main.py [hunk 3]
```diff
@@ -459,7 +459,7 @@ def check_for_updates(self):
         with open(Path(__file__).parent / 'last_update.txt', 'r') as f:
             local_last_update = f.read().rstrip()
 
-        latest_update = requests.get('https://raw.githubusercontent.com/RhinoSecurityLabs/pacu/master/last_update.txt').text.rstrip()
+        latest_update = requests.get('https://raw.githubusercontent.com/RhinoSecurityLabs/pacu/master/pacu/last_update.txt').text.rstrip()
```
Comment (RyanJarv) [inline pacu/main.py:462]: This of course got moved once the other PR was merged.

File: scripts/publish.sh [hunk 4]
```diff
@@ -21,4 +21,4 @@ fi
 
 ver="$(poetry version -s)"
 git tag "v${ver}"
-git push --tags
+git push origin "refs/tags/v${ver}"
```
Comment (RyanJarv) [inline scripts/publish.sh:24]: Tags pushes all tags, which isn't really what we want.

**Review by DaveYesland — APPROVED** [review]

**Pull request merged**